openlibrary/components/LibraryExplorer/components/OLCarousel.vue

```diff
@@ -251,7 +255,6 @@ export default {
 };
 </script>
 
-
 <style scoped>
```

Comment:
Please try to keep the changeset small by avoiding whitespace changes like this one.